linodecli/api_request.py

```diff
@@ -199,6 +203,47 @@ def _build_request_url(ctx, operation, parsed_args) -> str:
     return result
 
 
+def _traverse_request_body(o: Any) -> Any:
+    """
+    This function traverses is intended to be called immediately before
+    request body serialization and contains special handling for dropping
+    keys with null values and translating ExplicitNullValue instances into
+    serializable null values.
```

Comment:
I originally wanted to implement this as a JSONEncoder but the implementation ended up super cluttered.